### PR TITLE
ARTP-1175: Fix PWA header to switch themes correctly

### PR DESCRIPTION
**Pull request by rafgarciaa**
ezgif com-video-to-gif (3)

### CODE REVIEW

File: src/client/src/apps/StyleguideRoute/styled/index.tsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+export * from './Block'
 export { Paragraph } from './Paragraph'
-export type { ParagraphProps } from './Paragraph'
+export * from './Paragraph'
 export { Text, mapTextProps } from './Text'
-export type { TextProps } from './Text'
+export * from './Text'
 export { mapMarginPaddingProps } from './mapMarginPaddingProps'
-export type { MarginPaddingProps, MarginProps, PaddingProps } from './mapMarginPaddingProps'
+export * from './mapMarginPaddingProps'
```
Comment (rikoe) [inline src/client/src/apps/StyleguideRoute/styled/index.tsx:9]: Why are we changing these to star exports @rafgarciaa ? Isn't that frowned upon?

File: src/client/src/rt-theme/index.ts [hunk 2]
```diff
@@ -1,15 +1,8 @@
...
-  ColorPalette,
-  AccentName,
-  AccentPaletteMap,
-  Color,
-} from './colors'
-export { themes, getThemeColor } from './themes'
-export type { Theme, TouchableIntentName, ColorProps, ThemeSelector } from './themes'
+export * from './colors'
+export * from './themes'
+export * from './themes'
```
Comment (rikoe) [inline src/client/src/rt-theme/index.ts:5]: Same as above - is it more explicit to export specific types?

File: src/client/src/apps/StyleguideRoute/styled/index.tsx (hunk 1 above)
Comment (rafgarciaa) [reply src/client/src/apps/StyleguideRoute/styled/index.tsx:9]: I'm getting TS errors for some reason

File: src/client/src/rt-theme/index.ts (hunk 2 above)
Comment (rafgarciaa) [reply src/client/src/rt-theme/index.ts:5]: Same here

**Pull request merged**